pkg/asset/installconfig/powervs/client.go

```diff
@@ -515,3 +516,24 @@ func (c *Client) GetAuthenticatorAPIKeyDetails(ctx context.Context) (*iamidentit
 func (c *Client) GetAPIKey() string {
 	return c.APIKey
 }
+
+// GetVPCs gets all VPCs in a region
+func (c *Client) GetVPCs(ctx context.Context, region string) ([]vpcv1.VPC, error) {
+	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
```

Comment:
The installer recreates the context:
`ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)`